### PR TITLE
[13.0][MIG] sale_order_tag: Migration to 13.0

### PR DESCRIPTION
**Pull request by JSeguiN97**
Migration to 13.0

### CODE REVIEW

File: sale_order_tag/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "depends": ["crm", "sale"],
+    "data": [
+        "data/sale_order_tag_data.xml",
+        "security/ir.model.access.csv",
+        "security/sale_order_tag_security.xml",
+        "views/sale_order_tag.xml",
+        "views/sale_order.xml",
+    ],
+    "application": False,
+    "development_status": "Beta",
```
Comment (rousseldenis) [inline sale_order_tag/__manifest__.py:21]: Is this still Beta ?

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by danimv5 — APPROVED** [review]
Functional review! LGTM!!

**Comment by brendapaniagua** [comment]
Functional review. LGTM!

**Pull request merged**